src/main/java/com/patchworkmc/event/EventHandlerScanner.java

```diff
@@ -78,20 +78,6 @@ public AnnotationVisitor visitAnnotation(String annotation, boolean visible) {
 				throw new IllegalArgumentException("Methods marked with @SubscribeEvent must not be private, but " + name + " has private access");
 			}
 
-			if ((access & Opcodes.ACC_STATIC) == 0) {
```

Comment:
Should also remove the now-unused `isValidParentClass` method as well as the `superName` and `interfaces` fields